indexing-service/src/main/java/io/druid/indexing/overlord/HeapMemoryTaskStorage.java

```diff
@@ -200,6 +201,38 @@ public void addLock(final String taskid, final TaskLock taskLock)
   }
 
   @Override
+  public void setLock(String taskid, TaskLock taskLockToSet, TaskLockToggle taskLockToggle)
```

Comment:
Yes it is used only for logging and I think I can remove that parameter altogether